### ISSUE TITLE
Localise images linked from S3

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-cloud#257]
In `logs.md` we have some images that are linked from S3. These should be moved into the docs/assets directory and the URLs updated. In addition they should have alt text added.
